java-support/src/main/scala/io/cloudstate/javasupport/CloudStateRunner.scala

```diff
@@ -79,15 +79,16 @@ final class CloudStateRunner private[this] (
     case (serviceName, factory) => serviceName -> factory(system)
   }.toMap
 
-  // TODO JavaDoc
+  /**
+   * Creates a CloudStateRunner from the given services. Use the default config to create the internal ActorSystem.
+   */
   def this(services: java.util.Map[String, java.util.function.Function[ActorSystem, Service]]) {
-    this(ActorSystem("StatefulService", {
-      val conf = ConfigFactory.load()
-      conf.getConfig("cloudstate.system").withFallback(conf)
-    }), services.asScala.toMap)
+    this(ActorSystem("StatefulService", CloudStateConfigHolder.defaultConfiguration()), services.asScala.toMap)
```

Comment:
It's the other constructor, that takes a config explicitly, that should be supported. The config gets passed to the actor system. It should also be the config that gets passed through to other places (could be retrieved from the actor system). Might be more obvious if there's a test that sets the passivation timeout in config?

Reply:
@pvlugter I hope my solution does the job and I did not find a better way.